browser/migrations/0019_auto_20171103_2147.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.1 on 2017-11-03 21:47
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
seems like this and the previous migration create position and delete it again. Just remove the migrations if that's the case.

Reply:
I removed all of the previous migrations